common/types/loadbalancer.go

```diff
@@ -225,7 +236,11 @@ func NewL3n4Addr(protocol L4Type, ip net.IP, portNumber uint16) (*L3n4Addr, erro
 	if err != nil {
 		return nil, err
 	}
-	return &L3n4Addr{IP: ip, L4Addr: *lbport}, nil
+
+	addr := L3n4Addr{IP: ip, L4Addr: *lbport}
+	log.Debugf("created new L3n4Addr %s", addr)
```

Comment:
here as well

Reply:
I don't see the disadvantage of having more debug logs - see my comments above.